openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object that includes specified record of the table. Otherwise returns the relevant error.
+    remote isolated function getRecordById(string tableName, string sys_id, string? sysparmDisplayValue = (), boolean? sysparmExcludeReferenceLink = (), string? sysparmFields = (), boolean? sysparmQueryNoDomain = (), string? sysparmView = (), string? accept = ()) returns json|error {
+        string  path = string `/api/now/table/${tableName}/${sys_id}`;
+        map<anydata> queryParam = {"sysparm_display_value": sysparmDisplayValue, "sysparm_exclude_reference_link": sysparmExcludeReferenceLink, "sysparm_fields": sysparmFields, "sysparm_query_no_domain": sysparmQueryNoDomain, "sysparm_view": sysparmView};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Updates the specified record with the request body.
+    #
+    # + tableName - Name of the table in which the record is located.
+    # + sys_id - Unique identifier of the record to update.
+    # + payload - Name-value pairs for the field(s) to update in the associated table in JSON format.
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields.
+    # + sysparmExcludeReferenceLink - Flag that indicates whether to exclude Table API links for reference fields
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmInputDisplayValue - Flag that indicates whether to set field values using the display value or the actual value.
+    # + sysparmQueryNoDomain - Flag that indicates whether to restrict the record search to only the domains for which the logged in user is configured.
+    # + sysparmView - UI view for which to render the data. Determines the fields returned in the response.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + xNoResponseBody - By default, responses include body content detailing the new record. Set this header to true in the request to suppress the response body.
+    # + return - If successful, returns a JSON object that includes updated record of the table. Otherwise returns the relevant error.
+    remote isolated function updateRecordById(string tableName, string sys_id, json payload, string? sysparmDisplayValue = (), boolean? sysparmExcludeReferenceLink = (), string? sysparmFields = (), boolean? sysparmInputDisplayValue = (), boolean? sysparmQueryNoDomain = (), string? sysparmView = (), string? accept = (), string? contentType = (), string? xNoResponseBody = ()) returns json|error {
```

Comment:
An update is doing against an id. Should we say it in the function name? The appropriate function name would be `updateRecord`.

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98